test/coverage.jl

```diff
@@ -31,12 +291,32 @@ expected="""
     # The debug output isn't deterministic yet, so we can't test it.
     # We should figure out why and make it deterministic.
     #
-    # println()
-    # println(actual)
-    # println()
-    # for (a, e) in zip(split(actual, '\n'), split(expected, '\n'))
-    #     @test a == e
-    # end
+    # This will help figure out why:
+        # println()
+        # println(actual)
+        # println()
+        # for (a, e) in zip(split(actual, '\n'), split(expected, '\n'))
```

Comment:
did you solve it 👀 ?

Reply:
No, not yet.  It is deterministic on my machine, then on the server it behaves differently.

Reply:
I wonder if it's something with the gensym state across OS's?